### PR TITLE
Toolbar redesign

### PR DESCRIPTION
**Pull request by akaDuality**
- Remove router, keep all spit controller's logic in one place
- Move toolbar code to base controller
- Make left sidebar full height. Right can be done at all (Xcode can, but we can't)

Before 
Screen Shot 2022-08-20 at 23 37 34

After
Screen Shot 2022-08-20 at 23 38 31

### CODE REVIEW

**Review by drucelweisse — APPROVED** [review]

File: Document/Sources/Document/Assets/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -4,7 +4,7 @@
 "trait.selected.description" = "Selected.";
 
 /* Format for the description of the selected element; param0: the description of the element */
-"trait.selected.format" = "Selected: %@";
+"trait.selected.format" = "Selected. %@";
```
Comment (drucelweisse) [inline Document/Sources/Document/Assets/en.lproj/Localizable.strings:7]: should be done in russian localization too

Comment (akaDuality) [reply Document/Sources/Document/Assets/en.lproj/Localizable.strings:7]: It's already correct, I've checked

**Comment by akaDuality** [comment]
Simlify traits

**Comment by akaDuality** [comment]
@drucelweisse I've done a lot of minor redesign here, review again please

**Review by drucelweisse — APPROVED** [review]

**Pull request merged**